module/zfs/zfs_ioctl.c

```diff
@@ -4888,6 +4906,35 @@ zfs_ioc_userspace_upgrade(zfs_cmd_t *zc)
 	return (error);
 }
 
+/*
+ * inputs:
+ * zc_name		name of filesystem
+ *
+ * outputs:
+ * none
+ */
+static int
+zfs_ioc_userobjspace_upgrade(zfs_cmd_t *zc)
```

Comment:
It would be great to use a new-style ioctl here (which takes the nvlists).  Since there are no arguments besides the fs name, the change should be minimal from what you have here, but it at least follows the "right" example.